lib/org/chromium/webidl/ast/Attribute.js

```diff
@@ -13,7 +13,18 @@ foam.CLASS({
     'org.chromium.webidl.ast.Typed',
   ],
 
+  imports: [ 'source? as ctxSource' ],
+
+  properties: [
+    {
+      name: 'source',
+    },
+  ],
+
   methods: [
+    function init() {
+      this.source = this.ctxSource;;
```

Comment:
Using a factory was not sufficient. The factory was not triggered after parsing.
Thus, if we used a factory, all of the nodes would get the last value that was set on the parser.